test/game_collection_test.rb

```diff
@@ -33,6 +33,16 @@ def test_it_returns_home_win_pct
     assert_equal 49.83, @game_collection.pct_of_total_games(:home_wins)
   end
 
+  def test_it_returns_away_win_pct
+    @game_collection.create_pct_data
+    assert_equal 34.45, @game_collection.pct_of_total_games(:away_wins)
+  end
+
+  def test_it_returns_tie_pct
+    @game_collection.create_pct_data
+    assert_equal 15.72, @game_collection.pct_of_total_games(:ties)
+  end
```

Comment:
I like this setup, we'll just have to make sure it gets run through stat tracker